src/store/form_store.ts

```diff
@@ -0,0 +1,35 @@
+import { create } from 'zustand'
+import { createJSONStorage, persist } from 'zustand/middleware'
+import { immer } from 'zustand/middleware/immer'
+import { FormStore, FormStoreState } from '@/types/stores'
+
+const initialFormState: FormStoreState = {
+  description: '',
+  main_category: '',
+  sub_category: '',
+  coordinates: [0, 0],
+  email: '',
```

Comment:
telefoon en email waren hiervoor undefined en daardoor kon je ze 'leeg' laten in het formulier. nu wordt er een waarde gedetecteerd waardoor ik ze niet mee leeg kan laten